focus-group.js

```diff
@@ -13,6 +13,10 @@ function focus(current, next) {
 }
 
 function findGroupNodeByEventTarget(target) {
+  if (target.hasAttribute('focusgroup')) {
+    return target.closest('[focusgroup]')
```

Comment:
It is toooo limited support.

For instance, we need to support `inline` vs `block` (<kbd>←</kbd><kbd>→</kbd> vs <kbd>↓</kbd><kbd>↑</kbd>).